apps/studio/src/hooks/useIsUserIsomerUsers.ts

```diff
@@ -0,0 +1,29 @@
+import { useFeatureValue } from "@growthbook/growthbook-react"
+
+import type { GrowthbookIsomerUsersFeature } from "~/lib/growthbook"
+import { useMe } from "~/features/me/api"
+import {
+  ISOMER_ADMINS_AND_MIGRATORS_FEATURE_KEY,
+  ISOMER_ADMINS_FEATURE_KEY,
+} from "~/lib/growthbook"
+
+interface UseIsUserIsomerUsersProps {
+  includeMigrators: boolean
+}
+
+export const useIsUserIsomerUsers = ({
+  includeMigrators,
+}: UseIsUserIsomerUsersProps) => {
+  const {
+    me: { email },
+  } = useMe()
+
+  const { users } = useFeatureValue<GrowthbookIsomerUsersFeature>(
+    includeMigrators
+      ? ISOMER_ADMINS_AND_MIGRATORS_FEATURE_KEY
+      : ISOMER_ADMINS_FEATURE_KEY,
```

Comment:
question: hmm any reason why we decided to split into two feature flags? I thought the current design is quite extensible whereby the `users` key is the admins, then we can create a new key `migrators` (or can be called `guests`, since we can consider interns in the future) within the same feature flag.

Right now means we would have to duplicate the list of Isomer admins.

Reply:
ah no particular reason, i didn't realize growthbook config is a JSON - i tot it returns an array that we pass into `users`. Closed this PR and created a new one https://github.com/opengovsg/isomer/pull/1252